.github/workflows/example_notebook_servers_publish.yaml

```diff
@@ -0,0 +1,107 @@
+name: Build & Publish Notebook Servers
+on:
+  push:
+    branches:
+      - master
+      - v*-branch
+    paths:
+      - components/example-notebook-servers/**
+      - releasing/version/VERSION
+
+jobs:
+  base_images:
+    name: Build & Push - Base
+    uses: ./.github/workflows/example_notebook_servers_publish_TEMPLATE.yaml
+    secrets: inherit
+    with:
+      build_arch: linux/amd64,linux/arm64
+      image_folders: |
+        base
+
+  codeserver_images:
+    name: Build & Push - Code Server
+    uses: ./.github/workflows/example_notebook_servers_publish_TEMPLATE.yaml
+    needs: [ base_images ]
+    secrets: inherit
+    with:
+      build_arch: linux/amd64,linux/arm64
```

Comment:
shouldn't we instead use `linux/arm64/v8` here for M2 architecture?

Reply:
That is the implied default, and I don't know why we are explicitly specifying it as v8 in the other ones.

Reply:
@kimwnasptd Either way, I have never had an issue with images that are built for `linux/arm64`, and I actually think that `v7` is 32bit.